lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -165,23 +166,31 @@
                     url: this.options.disconnectUrl,
                     data: data,
                     dataType: 'html',
-                    success: function () {
+                    success: function() {
                         view.options.connected = false;
                         view.render();
                         view.showSuccessMessage();
                     },
-                    error: function (xhr) {
+                    error: function(xhr) {
                         view.showErrorMessage(xhr);
                     }
                 });
             },
 
             inProgressMessage: function() {
-                return this.indicators.inProgress + (this.options.connected ? gettext('Unlinking') : gettext('Linking'));
+                return HtmlUtils.interpolateHtml(
```

Comment:
Should this be `StringUtils.interpolate()` or should this function be `inProgressMessageHtml`?  I think the former.